runtime-admin/hexagonrpc/spec

```diff
@@ -0,0 +1,4 @@
+VER=0.3.2
```

Comment:
Add a postinst to:

- Create the sysusers.
- Reload the udev rules.

You might also want a debconf template to prompt the user to enable certain services, or even enabling them automatically.